app/org/sagebionetworks/bridge/play/controllers/StudyController.java

```diff
@@ -81,10 +89,15 @@ public Result getStudy(String identifier) throws Exception {
         return ok(Study.STUDY_WRITER.writeValueAsString(study));
     }
 
-    public Result getAllStudies() throws Exception {
+    public Result getAllStudies(String format) throws Exception {
         getAuthenticatedSession(ADMIN);
```

Comment:
I think we had previously decided that the researcher UI needs to be able to get a list of studies without being authenticated. So this getAuthenticatedSession(ADMIN) probably needs to go below the if block.